src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,25 @@
-export const TodoInfo = () => {};
+import { UserInfo } from '../UserInfo';
+import usersFromServer from '../../api/users';
+
+interface Todo {
+  id: number;
+  title: string;
+  userId: number;
+  completed: boolean;
+}
+
+export const TodoInfo = ({ todo }: { todo: Todo }) => {
+  const userInfo = usersFromServer.find(user => user.id === todo.userId);
+
+  return (
+    <article
+      data-id={todo.id}
+      className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+      <UserInfo
+        user={userInfo ? { email: userInfo.email, name: userInfo.name } : null}
+      />
```

Comment:
just add the check here not to render the component at all if the user doesn't exist
```suggestion
      {userInfo && <UserInfo
        user={userInfo}
      />}
```